subscribie/themes/theme-jesmond/jesmond/choose.html

```diff
@@ -110,11 +110,11 @@ <h5 class="body-lg">{{ plan.title|safe }}</h5>
                   </div>
                 {% endif %}
                 {% if plan.choice_groups %}
-                  <a href="{{ url_for('views.set_options', plan_uuid=plan.uuid) }}" class="btn-outlined-lg btn-block">
+                  <a href="{{ url_for('views.set_options', plan_uuid=plan.uuid) }}" class="btn-outlined-lg btn-block" name="{{ plan.title|safe}}">
```

Comment:
@joeltejeda nice workaround for targeting 👌
🚨Is it okay to avoid using `|safe`? Because using it turns off Jinja's escaping security 
https://jinja.palletsprojects.com/en/3.0.x/templates/#working-with-manual-escaping